lib/index.ts

```diff
@@ -79,7 +90,11 @@ async function buildDepTreeFromFiles(
   if (_.endsWith(lockFilePath, 'package-lock.json')) {
     lockFileType = LockfileType.npm;
   } else if (_.endsWith(lockFilePath, 'yarn.lock')) {
-    lockFileType = LockfileType.yarn;
+    if (fs.existsSync(path.resolve(root, lockFilePath.replace('yarn.lock', '.yarnrc.yml')))) {
+      lockFileType = LockfileType.yarn2;
+    } else {
+      lockFileType = LockfileType.yarn;
```

Comment:
Note: we need a way to send some meta back to track yarn v2 tests vs others

Reply:
The lock file is sent back in the return value of `buildDepTree`. You mean you need to handle the different type in the code that uses that library?